praw/models/reddit/comment.py

```diff
@@ -357,3 +357,20 @@ def __init__(self, comment: Comment):
 
         """
         self.thing = comment
+
+    def show(self):
+        """Uncollapse a :class:`~.Comment` that has been collapsed by Crowd Control.
+
+        Example usage:
+
+        .. code-block:: python
+
+           # lock a comment:
+           comment = reddit.comment('dkk4qjd')
+           comment.mod.show()
+        """
+        url = API_PATH["show_comment"]
+        return (
```

Comment:
Does this return anything useful? If not let's drop the return. If so, let's document the return.